requirements.txt

```diff
@@ -1,35 +1,48 @@
+appdirs==1.4.3
 appnope==0.1.0
 backports.shutil-get-terminal-size==1.0.0
 coverage==4.1
 coveralls==1.1
 decorator==4.0.10
-dj-database-url==0.4.1
-Django==1.10
+dj-database-url==0.4.2
+dj-static==0.0.6
+Django==1.10.7
 django-filter==0.14.0
+django-grappelli==2.8.2
 django-hashid-field==0.1.6
 django-toolbelt==0.0.1
 docopt==0.6.2
 factory-boy==2.7.0
 fake-factory==0.7.2
 flake8==3.0.4
 gnureadline==6.3.3
-gunicorn==19.6.0
+graphene==1.3
 graphene-django==1.0
+graphql-core==1.0.1
+graphql-relay==0.4.5
+gunicorn==19.7.1
+hashids==1.2.0
 ipython==4.2.0
 ipython-genutils==0.1.0
-mccabe==0.5.2
-pexpect==4.1.0
-pickleshare==0.7.2
-psycopg2==2.6.1
+iso8601==0.1.11
+mccabe==0.5.3
+packaging==16.8
+pexpect==4.2.1
+pickleshare==0.7.4
+promise==1.0.1
+psycopg2==2.7.1
 ptyprocess==0.5.1
 pycodestyle==2.0.0
 pyflakes==1.2.3
+pyparsing==2.2.0
 python-dateutil==2.5.3
 python-env==1.0.0
-requests==2.10.0
+pytz==2017.2
+requests==2.13.0
 simplegeneric==0.8.1
+singledispatch==3.4.0.3
 six==1.10.0
 static3==0.7.0
 traitlets==4.2.1
-whitenoise==3.2
-django-grappelli==2.8.2
+typing==3.6.1
+whitenoise==3.3.0
```

Comment:
@kola-er I didn't add those individually, they are requirements of another dependency.